vzlogger_buildroot_builder.Dockerfile

```diff
@@ -0,0 +1,236 @@
+# Dockerfile to build a buildroot-based cross-compile environment for vzlogger
+
+# the OS version used here is largely irrelevant,
+# as buildroot compiles it's own toolchain
+ARG	DEBIAN_VERSION=buster
+FROM	debian:$DEBIAN_VERSION as builder
+
+LABEL	maintainer="Thorben T. <r00t@constancy.org>"
+
+# don't inherit upstream labels
+LABEL	org.label-schema.build-date=null
+LABEL	org.label-schema.license=null
+LABEL	org.label-schema.name=null
+LABEL	org.label-schema.schema-version=null
+LABEL	org.label-schema.vendor=null
+
+# update host-OS and install required host packages
+# (we should optimize by using a base image that already contains this.)
+RUN	set -xe ; \
+	apt-get update ; \
+	apt-get -y --force-yes upgrade ; \
+	apt-get -y --force-yes install \
+		# https://buildroot.org/downloads/manual/manual.html#requirement
+		make gcc g++ \
```

Comment:
that's what my comment above says,
i only had not done any research yet,
thanks for the suggestion of the 'gcc' image(s).

Reply:
the gcc image itself does not make any sense,
because it's based on an image that already contains gcc,
which is then used to build gcc from source :rofl: 
https://github.com/docker-library/gcc/blob/master/10/Dockerfile

but we might use the image that the gcc image is build on:
it has most of what we need (still not all, but unlikely to get a perfect match).
https://github.com/docker-library/buildpack-deps/blob/master/debian/buster/Dockerfile

there is an existing official docker builds that uses buildroot to build uLibc usinbg buildroot,
similar to what i do here,
but is doesn't look like something we can re-use:
https://github.com/docker-library/busybox/blob/master/stable/uclibc/Dockerfile.builder